### DISCUSSION TITLE
在原内容 上方 显示翻译内容

### DISCUSSION THREAD

**Discussion by chanmazi** [immersive-translate/immersive-translate#379] [Ideas]

**Comment by theowenyoung** [comment]
这样好像有点奇怪... 正常不是 先原文 再译文吗？

**Comment by mrLuyao** [comment]
> 这样好像有点奇怪... 正常不是 先原文 再译文吗？

正常来说确实是先原文后译文，出版的书籍也是这样的。但我在浏览性阅读的时候，也会先去看译文，觉得译文不对劲才会去看原文，译文流畅的话就直接跳过原文了。而重点在学习语言的时候，会先看原文，看不懂才去看译文。

**Comment by trionline1234** [comment]
it will be better for language learner when listen bilingual text with translated text first, because the user can see image when read translated text first before listen original text.
for example
https://youtu.be/QllKhDdgY-4
